obspy/taup/tau.py

```diff
@@ -165,6 +165,56 @@ def copy(self):
         return self.__class__(super(Arrivals, self).copy(),
                               model=self.model)
 
+    def timeplot(self, phases, ax=None, show=True):
+        """
+        Method to plot the travel times of arrivals class, if any have
+        been calculated.
+
+        :param phases: Phases to plot.
+        :type phases: list of str
+        :param ax: Axes to plot to. If not given, a new figure with an axes
+            will be created. Must be a polar axes for the spherical plot and
+            a regular one for the Cartesian plot.
+        :type ax: :class:`matplotlib.axes.Axes`
+        :param show: Show the plot.
+        :type show: bool
+
+        :returns: The (possibly created) axes instance.
+        :rtype: :class:`matplotlib.axes.Axes`
+
+        """
+        import matplotlib.pyplot as plt
+        cmap = plt.cm.Dark2
+        colors = cmap(np.linspace(0, 1, len(phases)))
+        if not self:
+            raise ValueError("Can only plot calculated travel times.")
+
+        # create an axis/figure, if there is none, yet:
+        if not ax:
+            plt.figure(figsize=(10, 10))
+            ax = plt.subplot(111)
+
+        # extract the time/distance for each phase, and for each receiver:
+        for arrival in self:
+            phase = arrival.name
+            if phase in phases:
+                plt.plot(arrival.distance, arrival.time / 60, '.',
+                         label=phase, color=colors[phases.index(phase)])
+
+        # merge all arrival labels of a certain phase:
+        handles, labels = plt.gca().get_legend_handles_labels()
+        labels, ids = np.unique(labels, return_index=True)
+        handles = [handles[i] for i in ids]
+
+        plt.legend(handles, labels, loc=2, numpoints=1)
+
+        plt.grid()
+        plt.xlabel('Distance (degrees)')
+        plt.ylabel('Time (minutes)')
```

Comment:
ax.legend(), ax.grid(), ax.set_xlabel(), ...

Reply:
On it. How about existing instances in tau.py? For example, line 341. Do you want me to find and replace all of them?

Reply:
You don't have to touch stuff outside of the PR if you don't want to. But it looks like all lines that contain `plt.`-calls are covered (btw. [codecov github browser extensions](https://docs.codecov.io/docs/browser-extension) are really useful), so go ahead if you also want to fix those.